CHANGELOG.md

```diff
@@ -15,12 +15,14 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Bump `stackable-operator` to 0.87.0 ([#334]).
```

Comment:
We normally only document user-facing changes in here

Reply:
See https://github.com/stackabletech/airflow-operator/pull/591#discussion_r1979319141